web_decimal_numpad_dot/static/src/js/numpad_dot.js

```diff
@@ -9,7 +9,7 @@ odoo.define("web_decimal_numpad_dot.FieldFloat", function (require) {
     var NumpadDotReplaceMixin = {
         init: function () {
             this.events = $.extend({}, this.events, {
-                "keyup": "numpad_dot_replace",
+                "keydown": "numpad_dot_replace",
```

Comment:
I remember that this was changed to `keyup` on purpose due to other reason. Can you check? What happened when you keep the key pressed?

Reply:
Yes, I checked that. When you type fast, sometimes you hit another number before releasing the dot key, so we actually need it to be keydown. When you keep the key pressed, multiple keydown are sent, while only one keyup is sent at the end when you release it.